README.md

```diff
@@ -0,0 +1,95 @@
+# iprepd-nginx module
+
+`iprepd-nginx` is an openresty module for integrating with [iprepd](https://github.com/mozilla-services/iprepd).
+
+You can use the example configuration in this repo for a standalone proxy or install using [opm](https://github.com/openresty/opm)
+and integrate it yourself.
+
+*Note:* If nginx is behind a load balancer, make sure to use something like
+[ngx_http_realip_module](https://nginx.org/en/docs/http/ngx_http_realip_module.html).
+
+
+## Installation
+
+Install using [opm](https://github.com/openresty/opm)
+
+```
+opm get ajvb/iprepd-nginx
+```
+
+## Example
+
+Note: Check out `/etc` in this repo for a working example.
+
+```
+init_by_lua_block {
+  client = require("resty.iprepd").new({
+    url = os.getenv("IPREPD_URL") or "http://127.0.0.1:8080",
+    api_key = os.getenv("IPREPD_API_KEY"),
+    cache_ttl = os.getenv("IPREPD_CACHE_TTL") or 30,
+    threshold = tonumber(os.getenv("IPREPD_REPUTATION_THRESHOLD")),
```

Comment:
@ameihm0912 What do you think would be a good default or should I go for writing some guidance within the README?

Reply:
No hard rules for this; in the past we've used 50 (start warning/logging) and 25 (start blocking). But it is intended to be configurable for whatever the application wants to go with.

If we want defaults 50 is a good place to start here, but we should definitely document too because it should always be deliberately selected for any given app.